app/views/issues/new.html.slim

```diff
@@ -17,9 +17,19 @@ h1 Надіслати нове звернення
         = f.input :email, as: :email, label: 'Email',
                     placeholder: 'user@domain.com'
         = f.input :title, label: 'Короткий опис проблеми'
-        = f.association :category, collection: Category.order(:name).all, label: 'Категорія'
+        = f.association :category, collection: Category.order(:name).all,
+                    label: 'Категорія'
 	    = f.input :location, label: 'Адреса проблеми',
                     placeholder: 'Івано-Франківськ, Сахарова, 33'
         = f.input :description, as: :text, label: 'Текст звернення'
-        = f.input :attachment, as: :file, label: 'Додаток'
+        = f.input :attachment, as: :file, label: 'Додаток',
+                    accept: 'image/jpeg,image/gif,image/png'
```

Comment:
@simplews We have carrierwave gem, where we give the whitelist of allowed extensions.
We don't need this "accept")